### ISSUE TITLE
Only in Kiwi Browser tab freeze when working on Google Colab site

### ISSUE THREAD

**Issue by netrunner-exe** [kiwibrowser/src.next#608]
**Describe the bug**
Almost every day i work in Google Colaboratory. Only in the Kiwi browser, when I use Google Colab, the tab freezes when I try to expand the cell with code.  Google Colab is a interactive Jupyter notebook that is used for testing, writing and run code through a website (online).  This only happens in the Kiwi browser - Chrome, Firefox and Opera work fine.  Unfortunately, this is the only obstacle to remove Opera completely and switch to Kiwi because it is almost impossible to work in Google Colab due to tab freeze...

**To Reproduce**
I made test Colab notebook. Open link, tap to left arrow to expand cell - your tab will freeze. [Link to Google Colab](https://colab.research.google.com/drive/1VWv4r8YNW-LMivXMIh_53dFiZjgHYr13#scrollTo=A68TpC6YK9n-) 

**Expected behavior**
This is very annoying as I can't fully use my favorite browser.  This problem has been around for a long time, each time I hope that it will be fixed.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [e.g. Mi 9 Lite]
 - Android Version : [e.g. A11 ]
 - Kiwi Browser Version [ e.g. Latest ]

**Additional context**

**Comment by DUOLabs333** [comment]
Can you give us access?

**Comment by netrunner-exe** [comment]
> Can you give us access?

https://colab.research.google.com/drive/1VWv4r8YNW-LMivXMIh_53dFiZjgHYr13#scrollTo=A68TpC6YK9n-
Link with Colab example is with access for everyone, for using Colab just need to log in your Google account.

**Comment by netrunner-exe** [comment]
Example:
Expected working (Lemur browser for example, all works in other browsers too)
![ezgif-5-34e0b0ee07](https://user-images.githubusercontent.com/81887288/169018129-380c8fad-a280-44b9-9b65-3cdd1518e891.gif)

And Kiwi browser (after expanding the cell all tab with all elements freeze, browser without any extensions)
![ezgif-5-74e7ec34bb](https://user-images.githubusercontent.com/81887288/169018554-472d84ca-7d9a-4aa5-bb7f-809859a66a1d.gif)

**Comment by DUOLabs333** [comment]
I still can't access it.

**Comment by netrunner-exe** [comment]
> I still can't access it.

Sorry, i reenable acess, try https://colab.research.google.com/drive/1VWv4r8YNW-LMivXMIh_53dFiZjgHYr13?usp=sharing

**Comment by DUOLabs333** [comment]
I see the problem now.

**Comment by netrunner-exe** [comment]
> I see the problem now.

I hope this problem can be solved, thank you very much!

**Comment by netrunner-exe** [comment]
> It's especially weird, since all other browsers can handle it fine

No. I try to reinstall apk or without any extensions but it still freeze tab.

**Comment by gnadelwartz** [comment]
Sorry, can't help but confirm that google docs, tables etc. does not work well in kiwi. I'll contact Arnaud on discord https://discord.com/channels/494814211579445270/494885738547249153/984094566200254574

**Comment by hjyoung1** [comment]
It takes 10 - 15 sec to load for first time on kiwi for me. 
![kiwitest](https://user-images.githubusercontent.com/45882061/173197175-c380808e-a602-4b9f-a87c-298a5a0cbf4d.gif)

**Comment by DUOLabs333** [comment]
@netrunner-exe  Did the problem go away?
